### DISCUSSION TITLE
Can split the generated file ?

### DISCUSSION THREAD

**Discussion by goni098** [Brendonovich/prisma-client-rust#363] [Q&A] [answered]
My generated file is so large, it goes up to tens of thousands of lines and my editor becomes so lag . Can separate it to many files, maybe by each table ?

**Comment by Brendonovich** [comment]
This isn't currently possible, but shouldn't be too hard to implement. Could add a `clientFormat = "file" | "directory"` generator option.

**Reply by Brendonovich** [reply]
Will implement this today

**Reply by Brendonovich** [reply]
Got a basic version on `folder-generate-mode`

**Reply by goni098** [reply]
Can you explain it in more detail?

**Reply by Brendonovich** [reply, marked as answer]
Use `branch = "folder-generate-mode"` instead of `tag = ...`
Change `output` in the `generator` section to be a folder, not a file
Add `client_format = "folder"` to the `generator` section
Should work without a hitch, except for any differences with unreleased changes on `main`

**Reply by goni098** [reply]
thank you so much!

**Reply by chaozwn** [reply]
I've tested it, but it can't run properly.
